### ISSUE TITLE
Deprecation warning from node in event log

### ISSUE THREAD

**Issue by aldanor** [jhuckaby/Cronicle#38]
This is on node 7.10.1 and v0.7.3 of Cronicle. At the end of Shell Script task log, there's this:

```sh
(node:20574) DeprecationWarning: Calling an asynchronous function without callback is deprecated.
```

**Comment by jhuckaby** [comment]
Should be fixed in v0.7.4.  Thanks!

https://github.com/jhuckaby/Cronicle/releases/tag/v0.7.4

**Comment by aldanor** [comment]
Thank you!
